routes/video_routes.py

```diff
@@ -0,0 +1,107 @@
+#
+# @video_routes.py Copyright (c) 2021 Jalasoft.
+# 2643 Av Melchor Perez de Olguin, Colquiri Sud, Cochabamba, Bolivia.
+# <add direccion de jala la paz>
+# All rights reserved. #
+# This software is the confidential and proprietary information of
+# Jalasoft, ("Confidential Information"). You shall not
+# disclose such Confidential Information and shall use it only in
+# accordance with the terms of the license agreement you entered into
+# with Jalasoft.
+#
+
+from flask import Blueprint, request, jsonify
+from converters.video_to_video.video_to_video import VideoToVideoConverter
+from converters.video_to_images.video_to_images import VideoToImagesConverter
+from validators.VideoValidator import VideoValidator
+from helpers.endpoints_helper import save_file, get_or_save, update
+from helpers.compressor import FolderCompressor
+import os
+
+video_blueprint = Blueprint('video_routes', __name__)
+
+
+# Video Converter - Microservice
+@video_blueprint.route('/video-to-images', methods=['POST'])
+def video_to_images():
+    try:
+        video_path = save_file(request, 'file', 'video_to_frames_outputs')
+    except ValueError as e:
+        return jsonify({"error": e.args[0]}), 400
+
+    try:
+        file_in_db, file = get_or_save(video_path)
+    except Exception as e:
+        return jsonify({"error": f"Failed to save file to DB: {str(e)}"})
+
+    if file_in_db and file.output_path:
+        os.remove(video_path)
```

Comment:
I  suggest to delete this line, because it the file is finded in database, this will delete the video before processing it, turning into an error

Reply:
Done, thanks